tests/test_git_autoshare.py

```diff
@@ -7,8 +7,11 @@
 
 import pytest
 import yaml
-
-from git_autoshare.core import find_autoshare_repository, git_bin
+from git_autoshare.core import (
+    find_autoshare_repository,
+    find_wildcarded_repositories,
+    git_bin,
+)
```

Comment:
```suggestion

from git_autoshare.core import (
    find_autoshare_repository,
    find_wildcarded_repositories,
    git_bin,
)
```

Don't forget to install pre-commit.
I don't think the following of the unit tests will run if pre-commit fails.